server/public/styles.css

```diff
@@ -478,24 +532,170 @@ a:hover {
     border-radius: 5px;
   }
 
+  body {
+    font-size: 0.8rem;
+  }
+
   .font-size-10 {
+    font-size: 0.6rem;
+  }
+
+  .font-size-16 {
+    font-size: 0.57rem;
+  }
+
+  .flex-basis-40 {
+    flex-basis: 75%;
+  }
+
+  .column-half {
+    flex-basis: 70%;
+  }
+
+  .width-80 {
+    width: 75%;
+  }
+
+  .graph-button {
+    width: 28%;
+    font-weight: 800;
+    font-size: 0.6rem;
+  }
+
+  .graph-input {
+    width: 72%;
+    padding-left: 2px;
+    display: inline-block;
+  }
+
+  .button-font-size {
+    font-size: 10px;
+  }
+
+  .add-pr-button-font {
+    font-weight: 800;
     font-size: 15px;
   }
 
+  .button-height-37 {
+    display: inline-block;
+    height: 30px;
+  }
+
+  .font-size-20 {
+    font-size: 0.8rem;
+  }
+
+  .suggestions {
+    border: 1px solid #999;
+    border-top-width: 0;
+    list-style: none;
+    margin-top: 0;
+    max-height: 143px;
+    overflow-y: auto;
+    padding-left: 0;
+    width: calc(250px + 1rem);
+    position: absolute;
+    background-color: white;
+    border-radius: 5px;
+  }
+}
+
+@media only screen and (max-width: 360px) {
+  input {
+    border: 1px solid #999;
+    padding: 0.5rem;
+    width: 250px;
+    border-radius: 5px;
+  }
+
+  body {
+    font-size: 0.8rem;
+  }
+
+  .font-size-10 {
+    font-size: 0.6rem;
+  }
+
+  .font-size-16 {
+    font-size: 0.57rem;
+  }
+
   .flex-basis-40 {
     flex-basis: 75%;
   }
 
-  .flex-basis-20 {
-    flex-basis: 60%;
+  .column-half {
+    flex-basis: 70%;
   }
 
-  .flex-basis {
-    flex-basis: 15%;
+  .width-80 {
+    width: 75%;
+  }
+
+  .graph-button {
+    width: 28%;
+    font-weight: 800;
+    font-size: 0.6rem;
   }
 
-  .column-15 {
-    flex-basis: 30%;
+  .graph-input {
+    width: 72%;
+    padding-left: 2px;
+    display: inline-block;
+  }
+
+  .button-font-size {
+    font-size: 10px;
+  }
+
+  .add-pr-button-font {
+    font-weight: 800;
+    font-size: 15px;
+  }
+
+  .button-height-37 {
+    display: inline-block;
+    height: 30px;
+  }
+
+  .suggestions {
+    border: 1px solid #999;
+    border-top-width: 0;
+    list-style: none;
+    margin-top: 0;
+    max-height: 143px;
+    overflow-y: auto;
+    padding-left: 0;
+    width: calc(250px + 1rem);
+    position: absolute;
+    background-color: white;
+    border-radius: 5px;
+  }
+}
+
+@media only screen and (max-width: 411px) {
+  input {
+    border: 1px solid #999;
+    padding: 0.5rem;
+    width: 250px;
+    border-radius: 5px;
+  }
+
+  body {
+    font-size: 0.8rem;
+  }
+
+  .font-size-10 {
+    font-size: 0.6rem;
+  }
+
+  .font-size-16 {
+    font-size: 0.57rem;
+  }
+
+  .flex-basis-40 {
+    flex-basis: 75%;
```

Comment:
These don't sit right. A `.flex-basis-40` sometimes not being `flex-basis: 40%` is super confusing. You can use source order (from the CSS cascade) to help though.

1. sort your media queries in `min-width` ascending order.
1. define _new_ CSS classes for each size (where needed) with the correct number hint
    - `.flex-basis-40` in your earlier media query
    - `.flex-basis-75` in a later media query
1. apply _both_ CSS classes to your elements. the one later in your stylesheet will override the earlier one at the proper breakpoint

https://codepen.io/thebearingedge/pen/gOxajZb?editors=0100

https://user-images.githubusercontent.com/7432943/137406572-cded148c-6b2b-4756-9a8a-3665b5767ebd.mp4